_pages/typecheck.md

```diff
@@ -48,7 +48,7 @@ local b2: B = a1 -- not ok
 
 ## Builtin types
 
-The Luau VM supports 9 primitive types: `nil`, `string`, `number`, `boolean`, `table`, `function`, `thread`, `userdata`, and `buffer`. Of these, `table` and `function` are not represented by name, but have their dedicated syntax as covered in this [syntax document](syntax), and `userdata` is represented by [concrete types](#roblox-types); other types can be specified by their name.
+The Luau VM supports 10 primitive types: `nil`, `string`, `number`, `boolean`, `table`, `function`, `thread`, `userdata`, `vector`, and `buffer`. Of these, `table` and `function` are not represented by name, but have their dedicated syntax as covered in this [syntax document](syntax), and `userdata` is represented by [concrete types](#roblox-types); other types can be specified by their name.
```

Comment:
This implies that 'vector' can be used in type annotations - "other types can be specified by their name", but that's not true.
Outside of Roblox that defines 'Vector3' to refer to the VM builtin, default Luau configuration doesn't define a name for this type.

Reply:
```suggestion
The Luau VM supports 10 primitive types: `nil`, `string`, `number`, `boolean`, `table`, `function`, `thread`, `userdata`, `vector`, and `buffer`. Of these, `table` and `function` are not represented by name, but have their dedicated syntax as covered in this [syntax document](syntax), `userdata` is represented by [concrete types](#roblox-types), while `vector` is not representable by name at all; other types can be specified by their name.
```

What do you think of this? Or should I mention Roblox supports Vector3?

Reply:
We can skip the Roblox mention.
Addition looks ok and if we get more vector support out-of-the-box in the future, we will adjust it.